apps/auth/utils.py

```
@@ -0,0 +1,40 @@
+ALGORITHM = "HS256"
+
+def hash_password(password: str) -> str:
+    """Hash a password using bcrypt"""
+    salt = bcrypt.gensalt()
+    return bcrypt.hashpw(password.encode(), salt).decode()
+
+def verify_password(plain_password: str, hashed_password: str) -> bool:
+    """Verify a password against its hash"""
+    return bcrypt.checkpw(
+        plain_password.encode(),
+        hashed_password.encode()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)